independent-publisher-connectors/snowflake/apiDefinition.swagger.json

```diff
@@ -0,0 +1,745 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Snowflake REST",
+    "description": "Snowflake connector based on Snowflake REST service",

```

Comment:
@rekodus Can you please provide a detailed description for your connector, it should describe concisely the main purpose and value offered by your connector. Please follow the instructions mentioned in the certification documentation.

https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#write-a-description-for-your-connector